src/App.vue

```diff
@@ -42,6 +42,9 @@ export default {
     addTodo: function(){
       this.lists.push(this.comment)
     },
+    deleteList: function(i){
+      this.lists.splice(i,1)
```

Comment:
細かいですが、`,`のあとにはスペースを空けましょう。
```suggestion
      this.lists.splice(i, 1)
```